spec/fixtures/modules/task_param/tasks/win.ps1

```diff
@@ -0,0 +1,8 @@
+[CmdletBinding()]
+param(
+  [Parameter(Mandatory = $False)]
+  [String]
+  $_task
```

Comment:
One question I have in general about task metadata in the context of PowerShell.... would it make sense to pass in a single object containing key-value pairs, instead any one of x arbitrary parameters?

That would make it a lot easier to scaffold task templates from the PDK perspective, and reduces the need for future modifications to script prototypes, once a script accepts the metadata object.

Reply:
How do you envision people using them? A metadata hash, used like `meta['_task']`? We can talk about that a bit, but I'd like to get this fix out this week.

Reply:
Did you file a ticket for this at all @MikaelSmith ?

Reply:
I'll create a ticket, but without further input I'm not really in favor of this idea (and thus it probably won't go anywhere) for a couple reasons:
- it would require a change to the task specification, that affects how tasks use `_noop` in existing PE releases
- we'd want to name it something like `_meta` anyway to avoid collisions with valid user parameter names
- I'm not convinced it's that much of an improvement when they still need to go lookup what the metaparameter syntax is to be able to scaffold it, which means it doesn't feel worth the effort

Reply:
* I'm not sure how that would impact `_noop` if there are no PowerShell consumers of this concept yet?
* Only one thing to worry about colliding is better than more than one (agree that the risk of collision is probably low given I'm not aware of anyone who uses params in PowerShell that start with `_`)
* Never having to update PDK templates when new params are added is a long-term maintenance win (especially if the template comments a line like `Write-Debug $_meta | Select *` which will give you all the props)
* I think a single self-descriptive object is much better UX than receiving any number of arbitrary params (only passed when a script explicitly asks for them) - I'd wager that has the benefit of providing better flexibility for task authors running against multiple Bolt / task versions as well
* To expand on above, if a script is designed to run local, but can also receive task metadata, it's cleaner to have a single arg exposed (for things like tab completion)
* Generally speaking, introducing parameter objects is a good refactor